src/topology/algebra/module/basic.lean

```diff
@@ -955,6 +969,13 @@ lemma range_coprod [module R₁ M₂] [module R₁ M₃] [has_continuous_add M
   range (f₁.coprod f₂) = range f₁ ⊔ range f₂ :=
 linear_map.range_coprod _ _
 
+lemma comp_fst_add_comp_snd [module R₁ M₂] [module R₁ M₃] [has_continuous_add M₃]
+  (f : M₁ →L[R₁] M₃) (g : M₂ →L[R₁] M₃) :
+  f.comp (continuous_linear_map.fst R₁ M₁ M₂) +
+  g.comp (continuous_linear_map.snd R₁ M₁ M₂) =
+  f.coprod g :=
```

Comment:
I'm not sure what's our indentation style for formulas like this one. Do we have one or they're formatted differently in different files?

Reply:
This is what I tend to use, I don't think we're uniform